build-system/tasks/visual-diff.js

```diff
@@ -440,12 +402,12 @@ async function generateSnapshots(percy, page, webpages) {
  * @param {!puppeteer.Page} page a Puppeteer control browser tab/page.
  * @param {!JsonObject} webpages a JSON objects containing details about the
  *     pages to snapshot.
- * @param {string} config Config being used. One of 'canary' or 'prod'.
  */
-async function snapshotWebpages(percy, page, webpages, config) {
+async function snapshotWebpages(percy, page, webpages) {
   for (const webpage of webpages) {
     const {url, viewport} = webpage;
-    const name = `${webpage.name} (${config})`;
+    // TODO(rsimha): Remove the (prod) suffix after #18009 is merged.
+    const name = `${webpage.name} (prod)`;
```

Comment:
What I was trying to suggest offline was to call `percy.snapshot()` twice, once with ```name = `${webpage.name} (prod)` ``` and once with `name = webpage.name`. This way, we'll create duplicate snapshots on Percy (with and without the `(prod)` suffix,) and a week from now we can remove the snapshots with the `(prod)` suffix - that way, in-flight PRs won't fail because they're missing the correct snapshot name

Your decision :)

Reply:
The bigger problem is the time spent waiting for the additional snapshots to be uploaded / processed. I don't think it's worth it. The few in-flight PRs can be manually approved, and I can do so.

I'm going to go ahead and remove the suffix.